src/inmanta/data/__init__.py

```diff
@@ -4634,6 +4634,49 @@ async def get_status_for(
         out = await cls.select_query(query, [env, model_version, rids], no_obj=True)
         return {ResourceIdStr(r["resource_id"]): ResourceState[r["status"]] for r in out}
 
+    @classmethod
+    async def get_status_for_v2(
```

Comment:
Perhaps a more informative name would be `get_status_for_version` for the other method, and plain `get_status_for` / `get_status` for this one? As I understand it this one gets the current status of a resource, while the other gets the status for a specific version of a resource.